Makefile

```diff
@@ -9,3 +9,7 @@ $(DEPS):
 clean:
 	rm -rf $(DEPS)
 .PHONY: clean
+
+cgo-gen:
+	c-for-go --ccincl --ccdefs --nostamp filecoin.yml
```

Comment:
should we update the `clean` step to remove the generated files?

Reply:
Yeah, that's a good idea.